File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/user/UpdateUserBody.java
package com.lattis.ellipse.data.network.model.body.user;

import com.google.gson.annotations.SerializedName;

public class UpdateUserBody {

    @SerializedName("properties")
    private UserBody userBody;

    public UpdateUserBody(UserBody userBody) {
        this.userBody = userBody;
    }

    @Override
    public String toString() {
        return "UpdateUserBody{" +
                "userBody=" + userBody +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/user/UpdateUserBody.kt
package com.lattis.data.entity.body.user

import com.google.gson.annotations.SerializedName

class UpdateUserBody(@field:SerializedName("properties")
                     private val userBody: UserBody) {

    override fun toString(): String {
        return "UpdateUserBody{" +
                "userBody=" + userBody +
                '}'.toString()
    }
}
